src/store/selectors/activities/index.ts

```diff
@@ -108,6 +109,30 @@ function addCreateMemberOperationToActivites(
   return [...activities, newActivity];
 }
 
+function addEditMemberOperationToActivities(
+  state: RahaState,
+  activities: Activity[],
+  operation: EditMemberOperation
+): Activity[] {
+  const creatorMember = getOperationCreator(state, operation);
+  const description = operation.data.full_name
+    ? "edited their name."
+    : "edited their profile.";
```

Comment:
What does "edited their profile" mean? Is it an error right now if that happens?

(Also, what sort of edits will EDIT_MEMBER operation support in the future? Does email and video fall under this?)

Reply:
Hmm.. I would say that right now it's unexpected, but not really an error. We could alternately just not create an activity if the operation doesn't have a new full name associated with it?

In the future, I don't think email will result in an EDIT_MEMBER operation, but changing the video could.

Reply:
I think this is fine, was just wondering!